acyclic/src-3/acyclic/plugin/PluginPhase.scala

```diff
@@ -53,6 +55,14 @@ class PluginPhase(
       )
   }
 
+  private val pkgAccumulator = new tpd.TreeAccumulator[List[tpd.Tree]] {
+    def apply(acc: List[tpd.Tree], tree: tpd.Tree)(using Context): List[tpd.Tree] =
+      foldOver(
+        if (tree.symbol.is(Package) && tree.symbol.fullName.toString != "<empty>") tree :: acc else acc,
+        tree
+      )
+  }
+
```

Comment:
I copied the pattern in `pkgObjectAccumulator` for this, but I'm not sure that the behavior is correct or efficient. The packages appear to be repeated many more times with the `foldOver` compared to `collect`. This is how the packages compare between this Scala 3 plugin and the Scala 2 plugin:

Scala 3
```+ acyclic.CycleTests.force.mutualcyclic 83ms  List()
options: List(acyclic:forcePkg)
packageObjects
stdPackages
        Pkg(List(forcepkg, cyclicpackage, a))
        Pkg(List(forcepkg, cyclicpackage, a))
        Pkg(List(forcepkg, cyclicpackage))
        Pkg(List(forcepkg))
        Pkg(List(forcepkg, cyclicpackage))
        Pkg(List(forcepkg, cyclicpackage))
        Pkg(List(forcepkg))
        Pkg(List(forcepkg, cyclicpackage, b))
        Pkg(List(forcepkg, cyclicpackage, b))
        Pkg(List(forcepkg, cyclicpackage, b))
        Pkg(List(forcepkg, cyclicpackage, b))
        Pkg(List(forcepkg, cyclicpackage))
        Pkg(List(forcepkg))
        Pkg(List(scala, annotation)) <-- idk where these are coming from
        Pkg(List(scala))             <-- or if they should be here at all
        Pkg(List(acyclic))
        Pkg(List(acyclic))
        Pkg(List(forcepkg, cyclicpackage))
        Pkg(List(forcepkg, cyclicpackage))
        Pkg(List(forcepkg))
        Pkg(List(forcepkg, cyclicpackage, a))
        Pkg(List(forcepkg, cyclicpackage, a))
```

Scala 2
```
+ acyclic.CycleTests.force.mutualcyclic 56ms  List()
options: List(acyclic:forcePkg)
packageObjects
stdPackages
        Pkg(List(forcepkg, cyclicpackage, b))
        Pkg(List(forcepkg, cyclicpackage, a))
        Pkg(List(acyclic))
        Pkg(List(forcepkg, cyclicpackage))
        Pkg(List(forcepkg, cyclicpackage, b))
        Pkg(List(forcepkg, cyclicpackage))
        Pkg(List(forcepkg, cyclicpackage, a))
        
        
 ```